Package version column example

TJ-G opened the pull request:
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/<reponame>/blob/master/CONTRIBUTING.md).

TODO: Check the above box with an 'x' indicating you've read and followed [CONTRIBUTING.md](https://github.com/ni/<reponame>/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Adding an example that shows how to use a Notebook to add a 'package version' column to the Systems Grid. This will allow you to show a column for 'DAQmx Version' or similar in the Systems Grid

### Why should this Pull Request be merged?

So that SystemLink users can learn to further customize their Systems Grid, based on this as a starting point.

### What testing has been done?

Manually tested against several packages on internal demo server

TJ-G committed:
Create PackageVersionExample.ipynb

Adding an example that shows how to use a Notebook to add a 'package version' column to the Systems Grid. This will allow you to show a column for 'DAQmx Version' or similar in the Systems Grid

TJ-G committed:
Update PackageVersionExample.ipynb

Updating some copy-pasta that I missed

TJ-G committed:
Update PackageVersionExample.ipynb

joshuaprewitt commented:
You may want to mention the following if they want to make the report available for other users.

If you want other users on your server to access this report, save the report to C:\ProgramData\National Instruments\Skyline\JupyterHub\notebooks\_shared\reports.

TJ-G committed:
Update PackageVersionExample.ipynb

Adding note on how to share the notebook with other users on the SystemLink Server, as per Josh's recommendation

TJ-G committed:
Update PackageVersionExample.ipynb

Minor tweak to the new note
